boards/arm/swan_r5/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+.. _swan_r5_board:
+
+Blues Swan R5
```

Comment:
For doc generation purpose, this name should match the name provided in .yaml file: "Blues Wireless Swan".
Also please take care to adjust number of ### for the underlining.